main.vnet.tf

```diff
@@ -3,21 +3,13 @@ module "avm_res_network_virtualnetwork" {
   version = "0.2.3"
 
   resource_group_name = var.resource_group.name
-  name                = local.vnet_name
+  name                = "vnet"
```

Comment:
a vnet called vnet it is strange...

Reply:
I'm hoping we remove vnet creation altogether in future releases...